application/src/main/java/uk/nhs/hee/tis/revalidation/dto/RevalidationSummaryDto.java

```diff
@@ -0,0 +1,23 @@
+package uk.nhs.hee.tis.revalidation.dto;
+
+import io.swagger.annotations.ApiModel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.lang.Nullable;
+import uk.nhs.hee.tis.revalidation.entity.DoctorsForDB;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@ApiModel(description = "Revalidation Summary For Listing Pages")
+public class RevalidationSummaryDto {
+
+  private DoctorsForDB doctor;
+  private String gmcOutcome;
+  @Nullable
+  Boolean syncEnd;
+
```

Comment:
I don't think a `syncEnd` is really part of this class based on what I've read in your useful `@ApiModel` description.  Based on what I know about [DoctorsForDB](https://github.com/Health-Education-England/tis-revalidation-recommendation/blob/main/application/src/main/java/uk/nhs/hee/tis/revalidation/dto/DoctorsForDbDto.java) I think it's true of that too.  Will you discuss?

Reply:
Good point, it's no longer needed in DoctorsForDb

Reply:
@JosephKelly I've updated the api model description to be more accurate, this class is purely used as a message payload so it was misleading for me to suggest it could be used to directly drive the listing pages